README.md

```diff
@@ -38,7 +53,74 @@ Daffodil is a frontend toolkit that intends to accomplish three things:
+4. Developers who are forward-thinking, have gone through a re-platforming, and want to deflect future re-platforming. 
+
+## What packages do I use? <a id="whatpackages"></a>
+For a basic Ecommerce site, you need all of the most commonly used packages (common packages), which are the following:
+- [@daffodil/cart](./libs/cart/README.md)
+- [@daffodil/category](./libs/category/README.md)
+- [@daffodil/geography](./libs/geography/README.md)
+- [@daffodil/navigation](./libs/navigation/README.md)
+- [@daffodil/product](./libs/product/README.md)
+- [@daffodil/core](./libs/order/README.md)
+- [@daffodil/driver](./libs/driver/README.md)
+
+**Note:** Both the `core` and `driver` packages are dependencies for most other packages. Developers do not interact with these packages directly.
+
+Referring to the aforementioned [use cases](#whowhydaff), here are some lists and explanations of recommended Daffodil packages.
+
+### **1. To explore and experiment with PWAs**<a id="pwapackages"></a>
+
+ If you don't have a project in mind and want to see something in Daffodil working, then use:
+- [@daffodil/core](./libs/order/README.md)
+- [@daffodil/driver](./libs/driver/README.md)
+- [@daffodil/product](./libs/product/README.md)
+
+### **2. To expand Ecommerce opportunities**<a id="angularecompackages"></a>
+
+If you want to expand your portfolio for more Ecommerce opportunities, then use the [common packages](#whatpackages) and optionally include:
+- [@daffodil/design](./libs/design/README.md) (optional)
+
+You won't need to use more than the aforementioned common packages(#pwapackages) until you have a specific business use or requirement. However, if you are design-focused, then you can either use your own design framework or use the `design` package.
+
+### **3. To improve the user experience for existing business sites**<a id="frontendpackages"></a>
+If you have an existing site, either your own or backed by a business, then there are a few working assumptions about the site: it has SEO and specific URLs, and the product pages need to show lists of related products.
+ So, in addition to the [common packages](#whatpackages), you may need the following packages:
+- @daffodil/external-router](./libs/external-router/README.md)
+- [@daffodil/seo](./libs/seo/README.md)
+- [@daffodil/related-products](./libs/related-products/README.md)****
+- [@daffodil/upsell-products](./libs/upsell-products/README.md)
+- [@daffodil/design](./libs/design/README.md)
+- [@daffodil/contact](./libs/contact/README.md)
+
+### **4. To prevent future re-platforming**<a id="replatformpackages"></a>
+If you want to prevent future re-platforming of your Ecommerce site, there are a few assumed advanced requirements. For example, the business has complex product configurations and has marketing email campaigns. So, in addition to the [common packages](#whatpackages) and the packages for [improving frontend UX](#frontendpackages), you may also need the following packages:
```

Comment:
```suggestion
If you want to prevent future re-platforming of your Ecommerce site, there are a few assumed advanced requirements. For example, the business has complex product configurations and has marketing email campaigns. So, in addition to the [common packages](#whatpackages) and the packages for [improving user experiences](#frontendpackages), you may also need the following packages:
```